workflows/push.yml

```diff
@@ -0,0 +1,12 @@
+on: push
+name: Main
+jobs:
+  pHP-CS-Fixer:
```

Comment:
```suggestion
  PHP-CS-Fixer:
```

Reply:
The Github migration tool did that, so I'm lowering it all (I guess that key should start lowercase, the uppercase name is described after)